adapters/smoot/smoottest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
nit: rename to simple-app-banner.json to easily differentiate between the simple-web-banner.json